docs/changelog/release-9.md

```diff
@@ -0,0 +1,78 @@
+# Darklang Release 9
+
+January 1, 2023
+
+## Language and Standard Library
+
+- Support short-circuiting `||` and `&&` expressions (initially behind a flag -
+  go to Settings to enable)
+  ([#4640](https://github.com/darklang/dark/pull/4640))
```

Comment:
Do we need to update docs now accordingly, or when they're fully merged in?

`langaugedetails.md` currently states:
> We support `&&` and `||` - they do not currently short-circuit but we intend
them to in the future.

Reply:
Yes, we should.

Also, they'll be out by default when we announce this, so maybe we shouldn't say the flag thing?

Reply:
Removed the bit about the flag.

I added some wording to Language Details about the short circuiting operators. is it too much?